app/models/school.rb

```diff
@@ -1,5 +1,5 @@
 require 'json'
-require 'rgeo/geos/geos_c_impl'
+# require 'rgeo/geos/geos_c_impl'
```

Comment:
I think this change creeped back into the PR, we should go back to requiring this if I remember right.

Reply:
Strange! Went back and fixed that up.